pkg/export/template.go

```diff
@@ -11,16 +11,30 @@ import (
 	"text/template"
 	"time"
 
+	"github.com/cli/cli/pkg/iostreams"
 	"github.com/cli/cli/utils"
 	"github.com/mgutz/ansi"
 )
 
-func parseTemplate(tpl string, colorEnabled bool) (*template.Template, error) {
+type Template struct {
+	io           *iostreams.IOStreams
+	tablePrinter utils.TablePrinter
+	template     *template.Template
+}
+
+func NewTemplate(io *iostreams.IOStreams) Template {
+	return Template{
+		io:           io,
+		tablePrinter: utils.NewTablePrinter(io),
```

Comment:
Could we instead lazily instantiate the tablePrinter instance when the `row` helper is first used? Since the table printer measures the terminal and whatnot, it could incur some syscalls, and I think it's best that we avoid doing that if the table isn't actually going to be written to.

Reply:
Good point. Done.